### DISCUSSION TITLE
Renogy batteries in parallel

### DISCUSSION THREAD

**Discussion by paulus-71** [Louisvdw/dbus-serialbattery#586] [Q&A]
Today I connected my battery bank (3x Renogy 100Ah 12v Smart in parallel)
I daisy chained them with ethernet cables as prescribed in the manual. If I connect the Bluetooth BT-2 dongle I can see all three batteries and individual charge state, cell voltage etc.

I wanted to connect the batteries and use the BT dongle at the same time.
I ordered a waveshare rs485 to usb dongle and a DSD TECH SH-RT102 rs485 repeater which has one input and two outputs

I had a big struggle finding the right pins from the RJ45 output from the battery. Previous descriptions didn’t work for me.

I found that pin 2 (orange) 12v, pin 4 (blue) ground, pin 6 (green) A pin 7(white brown) B are the correct outputs from the RJ45 connections in the Renogy battery

I connected the 12v to vcc in in the repeater and made a connection to the output side for the BT-2 but not for the RS485 to USB.

Then connecting everything to the Rpi with Venus OS. BT dongle is working and Venus OS is showing the Renogy(serial)

Unfortunately Venus OS showing only one battery 100Ah. Is there a possibility to show all three batteries?
Is it possible to change the capacity to 300Ah for the one BMS or doesn’t that make any sense?

The only solution I can think of is take three rs485  splitters and three rs485 to usb dongles and connect the spitters to every uplink RJ 45 port would that work?

Kind regards,

Paul

**Comment by mr-manuel** [comment]
Hi Paul,

currently it does not work out of the box. See https://github.com/Louisvdw/dbus-serialbattery/issues/142. If I remeber correctly someone was able to do it somehow, but I can't remeber where I read it.

Maybe he/she reads this :-)
